workflows/createtaxdb.nf

```diff
@@ -98,7 +103,7 @@ workflow CREATETAXDB {
                                 }
 
         PIGZ_COMPRESS_AA ( ch_aa_for_zipping.unzipped )
```

Comment:
I am not sure I follow what the `PIGZ_COMPRESS_AA` do here

Reply:
Can ignore for now, it's for DIAMOND